napari_allencell_annotator/view/i_viewer.py

```diff
@@ -10,7 +11,7 @@ def __init__(self):
         super().__init__()
 
     @abstractmethod
-    def add_image(self, image: np.ndarray) -> None:
+    def add_image(self, image: BioImage) -> None:
         pass
```

Comment:
change back to dask array

Reply:
*numpy

Reply:
changed back in https://github.com/aics-int/napari-allencell-annotator/pull/90/commits/5a1eac3472eb10cd133a20dd0b9a299845829cba